crates/hyperion/src/ingress/mod.rs

```diff
@@ -66,8 +66,27 @@ pub fn process_handshake(
     }
 }
 
+#[derive(Resource)]
+pub struct ServerPingResponse {
+    pub description: String,
+    pub max_players: u32,
+}
+
+impl Default for ServerPingResponse {
+    fn default() -> Self {
+        Self {
+            description: String::from(
+                "Getting 10k Players to PvP at Once on a Minecraft Server to Break the Guinness \
+                 World Record",
+            ),
+            max_players: 12_000,
+        }
+    }
+}
+
 fn process_status_request(
     mut packets: EventReader<'_, '_, packet::status::QueryRequest>,
+    ping_response_data: Res<ServerPingResponse>,
```

Comment:
This needs to explicitly state that there's a lifetime parameter to compile:

```suggestion
    ping_response_data: Res<'_, ServerPingResponse>,
```